main.py

```diff
@@ -1,7 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
+
+GENRE_NAMES = ["Western", "Action", "Dramma"]
+ACTORS = [
```

Comment:
Grobal variables allow data to be shared and used across multiple functions or modules. You don't need it here

Reply:
`actors` could be list of tuples, so you'd iterate by them as in the checklist example